etherman/interfaces.go

```diff
@@ -0,0 +1,7 @@
+package etherman
+
+import "github.com/ethereum/go-ethereum/common"
+
+type dataAvailabilitier interface {
```

Comment:
`dataAvailabilitier` is hard to spell and even more to pronounce in verbal comunication, that's why I'm suggesting this change if it is still semantically correct :slightly_smiling_face: WDYT?

```suggestion
type l2DataProvider interface {
```

Reply:
agree in that the name isnt great, but since DA is kind of a buzz word in the industry, I think we should follow it. But I can change it to `dataAvailabilityProvider`